src/praisonai/tests/unit/test_yaml_cli_backend.py

```
@@ -0,0 +1,189 @@
+        description: Refactor utils.py
+        expected_output: Refactored code
+"""
+    
+    from praisonai.agents_generator import AgentsGenerator
+    
+    with patch('praisonai.cli_backends.resolve_cli_backend') as mock_resolve:
+        mock_backend = MagicMock()
+        mock_resolve.return_value = mock_backend
+        
+        generator = AgentsGenerator(config=yaml_content)
+        agents, tasks = generator.create_agents_and_tasks()
```

Ruff (0.15.10)

[warning] Unpacked variable `tasks` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)